Userland/Libraries/LibM/math.h

```diff
@@ -57,10 +57,6 @@ __BEGIN_DECLS
 #define M_SQRT2f32 1.41421356237309504880f
 #define M_SQRT1_2f32 0.70710678118654752440f
 
-#define M_TAU 6.28318530717958647692
-#define M_DEG2RAD 0.01745329251994329576
-#define M_RAD2DEG 57.2957795130823208767
```

Comment:
`math.cpp` is using `M_TAU`, where is it now getting that from?

Reply:
It's getting them from `LibCore/ExtraMathConstants.h`, as mentioned in the PR description. I'm not sure what's the right thing to do here, but on foreign environments LibC, LibCrypt, LibM and LibPthread are unavailable because the host already provides these libraries, so putting these constants inside LibM makes them out of reach.

Reply:
oh, I've missed the include that was added. :+1: